### ISSUE TITLE
[p1] Sending asset-token fails to cover the case where the asset's symbol contains Arabic numerals

### ISSUE THREAD

**Issue by shilei-wang** [irisnet/irishub#1618]
**Describe the bug**
If asset's symbol contains Arabic numerals, then sending token will fail , as well as query account.

**To Reproduce**

```bash
iriscli asset issue-token --family=fungible --source=native --symbol=abc1 --name=name --initial-supply=10000 --from=v0 --chain-id=shilei-qa --fee=0.6iris --commit
```
1) send:
```bash
iriscli bank send --amount=200abc1 --fee=0.004iris --commit --chain-id=shilei-qa --from=v0 --to=****
```
ERROR: coin name is empty

2) query:
```bash
iriscli bank account <v0_addr>
```
ERROR: invalid coin expression: 200abc1-min

**Comment by zhangyelong** [comment]
Resolves: #1617

**Comment by shilei-wang** [comment]
confirm fixed
